actors/common/src/main/java/org/sunbird/metrics/actors/OrganisationMetricsActor.java

```diff
@@ -458,11 +465,15 @@ private void orgCreationMetrics(Request actorMessage) {
           metricsResponseGenerator(responseFormat, periodStr, getViewData(orgId, orgName));
       sender().tell(response, self());
     } catch (ProjectCommonException e) {
-      ProjectLogger.log("Some error occurs", e);
+      ProjectLogger.log(
+          "OrganisationMetricsActor:orgCreationMetrics ProjectCommonException occured "
+              + e.getMessage(),
+          e);
       sender().tell(e, self());
       return;
     } catch (Exception e) {
-      ProjectLogger.log("Some error occurs", e);
+      ProjectLogger.log(
+          "OrganisationMetricsActor:orgCreationMetrics Exception occured " + e.getMessage(), e);
```

Comment:
From:
"OrganisationMetricsActor:orgCreationMetrics Exception occured "

To:
"OrganisationMetricsActor:orgCreationMetrics: Generic exception in getting org creation data: "